### DISCUSSION TITLE
HA Integration with Alarmo

### DISCUSSION THREAD

**Discussion by sfritz85** [Dilbert66/esphome-dsckeybus#49] [General]
esphome-dsckeybus is a great solution for HA.  Thanks for all your efforts.  Just wondering how to integrate it with Alarmo.  I would like to use alarmo to monitor and control all the notifications and select zones etc. for alarm modes.  Is there a way to do this and have the original alarm system work?  

Any suggestions, examples, configs or ideas greatly appreciated.

**Comment by Dilbert66** [comment]
Sorry, I don't know anything about Alarmo. Never used it.
